pkg/lifecycle/terraform/terraformer.go

```diff
@@ -135,7 +155,12 @@ func (t *ForkTerraformer) Execute(ctx context.Context, release api.Release, step
 		<-t.Daemon.MessageConfirmedChan()
 	}
 
-	return nil
+	err = t.StateSaver(debug, t.FS, t.StateManager, dir)
+	if err != nil {
+		return errors.Wrapf(err, "persist terraform state")
+	}
+
+	return err
```

Comment:
this should be nil

Reply:
Either make this a `return nil` or `return errors.Wrapf(err, "persist terraform state")` and get rid of the conditional

```suggestion
	return errors.Wrapf(err, "persist terraform state")
```

`errors.Wrap(nil, "string) == nil`